packages/lesswrong/lib/rateLimits/utils.ts

```diff
@@ -5,7 +5,9 @@ import { getDownvoteRatio } from "../../components/sunshineDashboard/UsersReview
 import { forumSelect } from "../forumTypeUtils"
 import { userIsAdmin, userIsMemberOf } from "../vulcan-users"
 import { autoCommentRateLimits, autoPostRateLimits } from "./constants"
-import { AutoRateLimit, RateLimitComparison, RateLimitInfo, rateLimitThresholds, RecentKarmaInfo, RecentVoteInfo, TimeframeUnitType, UserKarmaInfo, UserKarmaInfoWindow } from "./types"
+import { AutoRateLimit, RateLimitComparison, RateLimitInfo, RecentKarmaInfo, RecentVoteInfo, TimeframeUnitType, UserKarmaInfo, UserKarmaInfoWindow } from "./types"
+
+export type RateLimitUser = UserKarmaInfo|DbUser
```

Comment:
This should be defined in the `types.ts` file in the same directory, along with all the other types used by the rate limit code